package.json

```diff
@@ -36,28 +36,28 @@
     "npmAlgorithm": "flat"
   },
   "dependencies": {
-    "can-compute": "^3.0.0",
+    "can-compute": "^3.0.4",
     "can-event": "^3.0.1",
-    "can-map": "^3.0.1",
-    "can-util": "^3.0.1",
-    "jquery": "~2.2.1"
+    "can-map": "^3.0.3",
+    "can-util": "^3.0.13",
+    "jquery": "2.x - 3.x"
```

Comment:
Anyone know why `jquery` is a dependency?  My guess is that this should be a `devDep`.